### ISSUE TITLE
Read metrics and queries from multiple files.

### ISSUE THREAD

**Issue by free31307468** [albertodonato/query-exporter#108]
Query export is an useful tool for monitoring our applications. We like it very much and want to introduce it to our teams.

According to the document, except that the ```databases``` can be read from other files, section of ```metrics``` and ```queries``` must be written in the same file, and query exporter can only read one configuration file at a time.

In our case, we will only set up one query export and share it with multiple teams. 
Different teams may use the same database, but ```queries``` and ```metrics``` will be different, this scenario will cause a large configuration file eventually.

Also, from the point of view of collaborative work, when someone needs to add or modify these settings, they must modify the same file which might not be so efficient and also cause conflict when integrating all the changes.

So, my suggestion is that is it possible to develop a new feature that read the metrics and queries section from other sources for better maintenance practice, like supporting reading the settings from the config.yml file?

**Comment by jlweb-rsf** [comment]
This would be a great enhancement from our point of view. We have multiple teams that want to create queries with the Query Exporter - having these all in one config file will be very hard to scale.

**Comment by albertodonato** [comment]
No, that would include the content of the file as an element of the list.

What is supported is combining the content of the files. so each metrcs file can contain
```
metrics:
   m1: ..
   m2: ...
```
and so on, and similarly for queries.

Then you would pass all files to the exporter
